jobs.md

```diff
@@ -154,12 +154,79 @@ condor_on vgcnbwc-worker-c125m425-8231.novalocal
+by closing the job queue so that new jobs will not start until the strike is
+over. The strike protocol is as follows:
+- Write a blog post. You can reuse
+  [past blog posts](https://github.com/galaxyproject/galaxy-hub/blob/ca8a80e38b62e36a518570f219624755fb9ba0ac/content/news/2023-09-11-climate-strike/index.md).
+- Notify users in advance. In the past, this
+  [has been done displaying banner on the main page](https://github.com/galaxyproject/galaxy-hub/blob/ca8a80e38b62e36a518570f219624755fb9ba0ac/content/bare/eu/usegalaxy/notices.md?plain=1#L1-L7).
+- Enforce the strike using a TPV rule (read below).
+
+We have written a TPV rule that takes advantage of
+[HTCondor's job deferral feature](https://htcondor.readthedocs.io/en/latest/users-manual/time-scheduling-for-job-execution.html)
+to enforce the strike. A copy of
+[the rule](https://github.com/usegalaxy-eu/infrastructure-playbook/blob/e431458b17ef85162b9c6357810b2e1681774347/files/galaxy/tpv/tool_defaults.yml#L53-L75)
+is available below.
+
+```yaml
+      - id: climate_strike
+        # delay jobs using HTCondor's job deferral feature
+        # https://htcondor.readthedocs.io/en/latest/users-manual/time-scheduling-for-job-execution.html
+        if: True
+        execute: |
+          from datetime import datetime
+
+          strike_start = datetime(2023,9,15,7,0)
+          strike_end = datetime(2023,9,15,19,0)
+
+          training_roles = (
+              [r.name for r in user.all_roles()
+               if not r.deleted and r.name in
+               ("training-bma231-ht23", "training-msc-tmr-ws23",
+                "training-bio00058m")]
+              if user is not None else []
+          )
+
+          now = datetime.now()
+          if strike_start <= now < strike_end and not training_roles:
+              entity.params["deferral_time"] = f"{int(strike_end.timestamp())}"
+              entity.params["deferral_prep_time"] = "60"
+              entity.params["deferral_window"] = "864000"  # 10 days
+```
+
+Before merging the rule to
+[tool_defaults.yml](https://github.com/usegalaxy-eu/infrastructure-playbook/blob/e431458b17ef85162b9c6357810b2e1681774347/files/galaxy/tpv/tool_defaults.yml)
+(as a list item under the key
+[`tools.default.rules`](https://github.com/usegalaxy-eu/infrastructure-playbook/blob/e431458b17ef85162b9c6357810b2e1681774347/files/galaxy/tpv/tool_defaults.yml#L23C8-L23C8)),
+make sure to:
+- Change the strike start and end time.
+- Training roles are not always cleaned up after trainings are over. Thus, it
+  is necessary to **modify the rule so that ongoing trainings are not
+  affected**.
+  - First, log-in to the
+    [TIAAS admin panel](https://usegalaxy.eu/tiaas/admin/). The password, at the
+    moment of writing this text, can be found
+    [here](https://github.com/usegalaxy-eu/infrastructure-playbook/blob/e431458b17ef85162b9c6357810b2e1681774347/files/galaxy/tpv/tool_defaults.yml). 
```

Comment:
Not pointing to the correct file. Was this intentional?

Reply:
No, it was not, thanks for pointing it out. The correct link is

```suggestion
    [here](https://github.com/usegalaxy-eu/infrastructure-playbook/blob/e431458b17ef85162b9c6357810b2e1681774347/secret_group_vars/all.yml). 
```